### DISCUSSION TITLE
Compensation of solar and wind

### DISCUSSION THREAD

**Discussion by holger-wi** [electricitymaps/electricitymaps-contrib#7535] [Ideas] [answered]
Hello,

it would be very interesting if the compensated power of solar and wind would could be visualized as well. If possible.

Regards
Holger-wi

**Comment by madsnedergaard** [comment, marked as answer]
Thanks for sharing, that would require the data to be available for that in a significant amount of regions, which I unfortunately doubt it will be any time soon :)
